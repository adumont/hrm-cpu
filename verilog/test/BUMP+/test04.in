25 9e
57 10
30 ea
25 78
08 38
45 8f
33 a3
29 18
08 0c
03 7e
42 49
05 f3
05 b5
22 da
40 92
40 c3
51 e1
21 be
01 79
31 b6
20 7d
37 72
18 b0
08 07
32 66
02 3d
20 39
03 01
32 e2
47 53
20 42
31 71
15 7f
55 9d
58 77
13 48
01 4c
15 23
48 9b
35 cb
01 c7
