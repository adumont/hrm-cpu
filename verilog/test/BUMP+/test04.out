9f
11
eb
79
39
90
a4
19
0d
7f
4a
f4
b6
db
93
c4
e2
bf
7a
b7
7e
73
b1
08
67
3e
3a
02
e3
54
43
72
80
9e
78
49
4d
24
9c
cc
c8
